src/index.jsx

```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import Routes from './router.jsx';
 import Career from './containers/career.jsx';
-import Style from '../public/style.css';
+import Style from '../public/scss/style.scss';
```

Comment:
wicked !
now you can get rid of the style.css ( not scss ) file cause it's just loads of extra code

Reply:
@Heathercoraje